presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -3246,9 +3372,6 @@ private void doInsert(HiveStorageFormat storageFormat, SchemaTableName tableName
                 assertEquals(otherTableStatistics.getRowCount().getAsLong(), CREATE_TABLE_DATA.getRowCount() * 3L);
             }
 
-            // verify we did not modify the table directory
-            assertEquals(listAllDataFiles(transaction, tableName.getSchemaName(), tableName.getTableName()), existingFiles);
```

Comment:
I assume this is fixed by https://github.com/prestodb/presto/commit/a507ab46a09398ab25c0045bd66b4350b88c0c3a